README.md

```diff
@@ -1,45 +1,231 @@
+		Crds: []*apiextensions.CustomResourceDefinition // optional: additional custom resource definitions to be installed
+    },
+    CheckFn: func (context.Context, *unstructured.Unstructured, *logr.Logger, ClusterInfo) (bool, error) { // optional: custom logic for resource state checks
+		return true, nil
+	},
+    CheckReadyStates: true,
+}
+
+// Based on deployInfo above following operations could be performed 
+
+// Option 1: Install resources
+ready, err := manifest.InstallChart(logger, deployInfo, []types.ObjectTransform{})
+if err != nil {
+	return false, err
+}
+
+// Option 2: Verify resources exist
+ready, err := manifest.ConsistencyCheck(logger, deployInfo, []types.ObjectTransform{})
+if err != nil {
+    return false, err
+}
+
+// Option 3: Uninstall resources
+ready, err := manifest.UninstallChart(logger, deployInfo, []types.ObjectTransform{})
+if err != nil {
+return false, err
+}
 ```
-  args = map[string]string{
-    "set": "xyz=abc",
-    "flags": "CreateNamespace=true,Namespace=new-space",
-  }
-  ```
+
+</details>
+
+## Run the operator 
+
+### Local setup
+
+- Refer to the local cluster and registry setup [documentation](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/developer/provision-cluster-and-registry.md#local-cluster-setup) to create cluster(s) either in a single-cluster or dual-cluster mode.
+- Set your `KUBECONFIG` environment variable to point towards the desired cluster.
+- Run the following `make` file commands:
+
+| Make command | Description                                          |
+|--------------|------------------------------------------------------|
+| build        | Run fmt, vet and DeepCopy method implementations     |
+| manifests    | Create RBACs CRDs based on [API types](operator/api) |
+| install      | Install CRDs                                         |
+| run          | Run operator controller locally                      |
+
+### Cluster setup
+
+- Refer to the remote cluster and registry setup [documentation](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/developer/provision-cluster-and-registry.md#remote-cluster-setup) to create cluster(s) either in a single-cluster or dual-cluster mode.
+- Set your `KUBECONFIG` environment variable to point towards the desired cluster.
+- Run the following commands with `IMG` environment variable pointing to the image name
+
+| Make command | Description                                           |
+|--------------|-------------------------------------------------------|
+| docker-build | Build operator image                                  |
+| docker-push  | Push docker image to your repo                        |
+| deploy       | Deploys the operator resources to the desired cluster |
+
+## Contribution
+Refer to [Kyma contribution guidelines](https://kyma-project.io/community/contributing/02-contributing/).
+
+## Versioning and Releasing
```

Comment:
We use [sentence case](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#sentence-case) in headlines. 

```suggestion
## Versioning and releasing
```